runtime/reference/node.md

```diff
@@ -182,13 +239,13 @@ of moving components in your project. Deno also has a more secure runtime, with
 [runtime permissions](../basics/permissions.md) that allow you to control what
 your code can access.
 
-#### deno.json (optional)
+#### deno.json
 
 Deno has its own config file, `deno.json` or `deno.jsonc`, which can be used to
 configure your project. You can use it to define tasks, dependencies, path
 mappings, and other runtime configurations.
 
-#### Migrating npm scripts to deno.json (optional)
+#### Migrating npm scripts to deno.json
 
 If preferred, you can move your npm scripts over to `deno.json`, where they can
 be run using `deno task`. This allows you to manage all necessary permission
```

Comment:
"If you're interested in only having a `deno.json` file, with a `package.json`, you can define easy to remember names for shell invocations. This is pretty much exactly like the `"scripts"` section of `package.json"

Reply:
I don't really understand the intent of this sentence.